### PR TITLE
add git stash indication

### PR DESCRIPTION
**Pull request by ev-agelos**
The icon doesn't look nice with the rest I just chose one :/
And I am not sure for the color too :D

### CODE REVIEW

File: plugins/git/plugin.zsh [hunk 1]
```diff
@@ -172,13 +176,17 @@ geometry_prompt_git_render() {
     fi
   fi
 
+  if git rev-parse --quiet --verify refs/stash >/dev/null; then
```
Comment (jedahan) [inline plugins/git/plugin.zsh]: I would check `PROMPT_GEOMETRY_GIT_SHOW_STASHES` here first

File: plugins/git/plugin.zsh [hunk 2]
```diff
@@ -29,6 +32,7 @@ PROMPT_GEOMETRY_GIT_CONFLICTS=${PROMPT_GEOMETRY_GIT_CONFLICTS:-false}
 PROMPT_GEOMETRY_GIT_TIME=${PROMPT_GEOMETRY_GIT_TIME:-true}
 PROMPT_GEOMETRY_GIT_TIME_LONG_FORMAT=${PROMPT_GEOMETRY_GIT_TIME_LONG_FORMAT:-false}
 PROMPT_GEOMETRY_GIT_TIME_SHOW_EMPTY=${PROMPT_GEOMETRY_GIT_TIME_SHOW_EMPTY:-true}
+PROMPT_GEOMETRY_GIT_STASHES=${PROMPT_GEOMETRY_GIT_STASHES:-false}
```
Comment (jedahan) [inline plugins/git/plugin.zsh]: can we name this `PROMPT_GEOMETRY_GIT_SHOW_STASHES` and set the default to `true` ?

File: plugins/git/plugin.zsh [hunk 3]
```diff
@@ -172,13 +176,17 @@ geometry_prompt_git_render() {
...
+  fi
+
   local render="$(prompt_geometry_git_symbol)"
 
   if [[ -n $render ]]; then
     render+=" "
   fi
 
-  render+="$(prompt_geometry_git_branch) ${conflicts}${GEOMETRY_GIT_SEPARATOR}${time} $(prompt_geometry_git_status)"
+  render+="$(prompt_geometry_git_branch) ${conflicts}${GEOMETRY_GIT_SEPARATOR}${time} $(prompt_geometry_git_status) ${stashes}"
```
Comment (jedahan) [inline plugins/git/plugin.zsh]: I'd like to see what this looks like before the status, with the other 'optional' parts.
Also we should include the space / GEOMETRY_GIT_SEPARATOR as part of the `stashes=` assignment so if stashes is unassigned we don't have a trailing space.

**Review by jedahan — APPROVED** [review]
Great addition! Do you have the time to update the PR with my suggestions or should I?

**Comment by jedahan** [comment]
Trying to find a replacement for the pin icon that keeps simple, solid geometry - these seem nice:

# `●` `✦` `╸` `⚈` 

I want to emphasize that stashes are not as big a deal as conflicts too

File: plugins/git/plugin.zsh (hunk 2 above)
Comment (ev-agelos) [reply plugins/git/plugin.zsh]: Sure!

File: plugins/git/plugin.zsh (hunk 1 above)
Comment (ev-agelos) [reply plugins/git/plugin.zsh]: Right. Ill update it

**Comment by ev-agelos** [comment]
I will update the PR with your suggestions, thanks for the review!
It is my 2nd contribution and I would like to to do it from start to finish :D

**Comment by jedahan** [comment]
testing today

**Comment by jedahan** [comment]
Thanks for your patience and the contribution!

**Comment by ev-agelos** [comment]
Awesome! Thanks a lot too for you guidance!

**Pull request merged**